steps/20/README.md

```diff
@@ -62,13 +63,16 @@ Additionally, we set the formatting option `showMeasure` to `false`. This hides
       items="{invoice>/Invoices}" >
       <items>
          <ObjectListItem
+            core:require="{
+               Currency: 'sap/ui/model/type/Currency'
+            }"
             title="{invoice>Quantity} x {invoice>ProductName}"
             number="{
                parts: [
                   'invoice>ExtendedPrice', 
                   'view>/currency'
                ],
-               type: 'sap.ui.model.type.Currency',
+               type: 'Currency',
```

Comment:
Can you please also update the description. There it still states "... then we apply the currency data type on the number by setting the `type` attribute of the binding syntax to `sap.ui.model.type.Currency`. "

Reply:
Done